.hound-rubocop.yml

```diff
@@ -234,9 +242,120 @@ Lint/HandleExceptions:
   Description: Don't suppress exception.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions
   Enabled: false
-Lint/LiteralInCondition:
+Lint/LiteralAsCondition:
   Description: Checks of literals used in conditions.
   Enabled: false
 Lint/LiteralInInterpolation:
   Description: Checks for literals used in interpolation.
   Enabled: false
+
+##################### Rails ##################################
```

Comment:
Is this relevant, this isn't a Rails project.

Reply:
Not relevant. I will delete.